pulsar-client-admin/src/main/java/org/apache/pulsar/client/admin/internal/PulsarAdminBuilderImpl.java

```diff
@@ -54,13 +55,24 @@ private PulsarAdminBuilderImpl(ClientConfigurationData conf) {
 
     @Override
     public PulsarAdminBuilder clone() {
-        return new PulsarAdminBuilderImpl(conf.clone());
+        PulsarAdminBuilderImpl pulsarAdminBuilder = new PulsarAdminBuilderImpl(conf.clone());
+        pulsarAdminBuilder.clientBuilderClassLoader = clientBuilderClassLoader;
+        pulsarAdminBuilder.acceptGzipCompression = acceptGzipCompression;
+        return pulsarAdminBuilder;
     }
 
     @Override
     public PulsarAdminBuilder loadConf(Map<String, Object> config) {
         conf = ConfigurationDataUtils.loadData(config, conf, ClientConfigurationData.class);
         setAuthenticationFromPropsIfAvailable(conf);
+        if (config.containsKey("acceptGzipCompression")) {
+            Object acceptGzipCompressionObj = config.get("acceptGzipCompression");
+            if (acceptGzipCompressionObj instanceof Boolean) {
```

Comment:
@nodece no need to shield against null since there's already `containsKey` check. explicitly putting a `null` value in the config wouldn't make any sense. That's why I think it's better to let the code throw NullPointerException in that case.